ThreeBotPackages/zerobot/admin/actors/notifications.py

```diff
@@ -8,4 +12,7 @@ def _init(self, **kwargs):
 
     @j.baseclasses.actor_method
     def check_new_release(self, schema_out=None, user_session=None):
-        return j.data.serializers.json.dumps(Tools.get_newest_version())
+        if Tools.is_latest_release():
```

Comment:
shouldnt this be if not latestversion?

Reply:
yes, have to be `not is_latest_release()`, changed and thanks Jo